apps/addresses/models.py

```diff
@@ -46,7 +49,7 @@ def __str__(self):
         return value
 
     def clone(self):
-        address_clone = PostalAddress.objects.create(
+        address_clone = BasePostalAddress.objects.create(
```

Comment:
This is not correct. I think It would need to be changed to something like `self.__class__.objects.create(...)`

Reply:
Done